### DISCUSSION TITLE
Open data to support the IRA's Energy Infrastructure Reinvestment Financing

### DISCUSSION THREAD

**Discussion by zaneselvans** [catalyst-cooperative/pudl#1849] [Ideas]
Based on the policy work we've supported so far, the most relevant part of the IRA seems to be the vastly expanded loan guarantee program and it's focus on "Energy Infrastructure Reinvestment Financing." This should help facilitate the retirement of existing uneconomic fossil infrastructure, and its replacement with zero carbon energy systems. Here's [a brief summary from the Congressional Research Service](https://crsreports.congress.gov/product/pdf/IN/IN11984):

> The IRA would also establish a time-limited (available through FY2026), $250 billion loan guarantee commitment authority for “Energy Infrastructure Reinvestment Financing.” As defined in the bill, energy infrastructure would include (1) electricity generation and transmission, or (2) production, processing, and delivery of fossil fuels, petroleum-derived fuels, or petrochemical feedstocks. Additionally, $5 billion would be appropriated to carry out the program. To qualify, projects would need to:
> 1. retool, repower, repurpose, or replace energy infrastructure that has ceased operations — subject to a requirement that fossil fuel electricity generation projects must avoid, reduce, utilize, or sequester air pollutants and anthropogenic greenhouse gas emissions, or
> 2. enable operating energy infrastructure to avoid, reduce, utilize, or sequester air pollutants or anthropogenic emissions of greenhouse gases.

![image](https://user-images.githubusercontent.com/596279/185756522-9f6b0bb9-efdb-42fe-b03e-01b4a619995b.png)

**This will be a crash program. All $250B must be deployed in the next 5 years.** The actual capital investment decisions are going to mostly take place in state utility commission proceedings. The tax credits offered elsewhere in the IRA should make these proceedings less adversarial than in the past, but there will still be lots of wrangling over who benefits from the transition: ratepayers, shareholders, labor, local tax jurisdictions, independent power producers, utilities, etc. And given the very high tax subsidies for CCS and CDR, there may be competition between cheap new clean energy and expensive retrofits of existing fossil generation that may or may not pan out.

## What open data will best support this policy work?

* Obviously getting the new FERC Form 1 XBRL data integrated and continuing to update and expand our coverage of the EIA 860/861/923 data is important. 
* The inclusion of petroleum and petrochemical infrastructure, and natural gas pipelines (serving electricity generation or otherwise) suggests that the FERC Form 2 and EIA Form 176, which we're already trying to integrate, will be valuable.
* The FERC Form 6 (which is like Form 1 & 2 but for oil) seems like it might also be relevant.
* Can this financing help support closing and remediating existing coal mines? Would the MSHA Mines data be helpful? Is there relevant USGS data that's currently difficult to access?

**Comment by zaneselvans** [comment]
Short of reading the relevant sections of the 730 page bill itself, does anybody have any favorite explainers / summaries of the Energy Infrastructure Reinvestment Financing policies?  I'll list whatever I find here:
* [Details from the US Senate Democrats](https://www.democrats.senate.gov/inflation-reduction-act-of-2022). Sadly the summaries are all very superficial.
* [Dave Roberts & Jesse Jenkins discussing the IRA](https://www.volts.wtf/p/diving-further-into-the-inflation-d7e#details)
* [Evergreen Action summary (15 pages)  of the whole bill](https://www.evergreenaction.com/documents/The-Climate-Impact-of-the-IRA.pdf)
* [NYTimes article about the loan program](https://www.nytimes.com/2022/08/22/business/energy-environment/biden-climate-bill-energy-loans.html)
* [Forbes column from Mike O'Boyle](https://www.forbes.com/sites/energyinnovation/2022/08/24/inflation-reduction-act-benefits-billions-in-just-transition-funding-for-coal-communities)
* [RMI Blog post on 1706](https://rmi.org/important-clean-energy-policy-youve-never-heard-about/)

**Comment by zaneselvans** [comment]
Some suggestions from [the Twitter thread](https://twitter.com/CatalystCoop/status/1561790305807962112):
* remaining plant balance (See: FERC 1 depreciation studies. Gotta get that integrated into the DB)
* plant employment (Need to disentangle the plant "total" rows and pull this out of the FERC 1 steam plants table)
* Infrastructure (interconnection, transmission, land, port access, etc) that supports fossil assets (across IOUs, co-ops and munis)
* balance sheet capacity and credit ratings of plant owners
* Local RE reinvestment potential
* Property taxes and other local revenue associated with the fossil assets or their owners

**Reply by swinter2011** [reply]
I've found a handful of law firms that provide helpful summaries of key energy provisions (and other provisions) in the IRA:
- [Holland and Knight summary](https://www.hklaw.com/-/media/files/insights/publications/2022/08/080822inflationreductionactsummary.pdf?la=fr)
- [McGuire Woods very readable summary](https://www.mcguirewoods.com/client-resources/Alerts/2022/8/us-senate-introduces-inflation-reduction-act)
- For project finance, [Norton Rose Fulbright](https://www.projectfinance.law/tax-equity-news/2022/august/qa-on-the-inflation-reduction-act/#a13) provides the most detailed summary.

The Bipartisan Policy Center also has a [good summary](https://bipartisanpolicy.org/blog/inflation-reduction-act-summary-energy-climate-provisions/)
